### PR TITLE
dts: Rename SM-G530Y from gprimeltezt to fortunaltezt

### PR DESCRIPTION
**Pull request by wonderfulShrineMaidenOfParadise**
v2: Rename to fortunaltezt ~~and add panel selection~~

### CODE REVIEW

File: dts/msm8916/msm8916-samsung-r11.dts [hunk 1]
```diff
@@ -20,6 +20,14 @@
 		compatible = "samsung,gprimeltezt", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "G530Y*";
 
+		panel {
```
Comment (TravMurav) [inline dts/msm8916/msm8916-samsung-r11.dts:23]: Do you really need it for a single entry here?

Comment (wonderfulShrineMaidenOfParadise) [reply dts/msm8916/msm8916-samsung-r11.dts:23]: Some of the Grand Prime e.g. SM-G530FZ/H/W variants have 2 panel entries but this one has only one.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
`gprimeltezt` stands for SM-G530MU in fact. Changing it to `fortunaltezt` to avoid confusions.

**Review by stephan-gh — APPROVED** [review]

**Comment by stephan-gh** [comment]
The commit message is very weird, we don't do v2 etc in lk2nd. Please just make the commit message like "dts: Rename SM-G530Y to fortunaltezt` or something like that.

**Review by TravMurav — APPROVED** [review]
while this rename is just a droplet into the mess that is "properly give compatibles to all devices in lk2nd for the future dtb selection", it's probably fine to move things a bit :)

**Pull request merged**